{"name": "Get order with invalid track â€” error", "status": "passed", "steps": [{"name": "Send GET with invalid track", "status": "passed", "start": 1755207681643, "stop": 1755207681736}, {"name": "Check status code and error message", "status": "passed", "start": 1755207681736, "stop": 1755207681736}], "start": 1755207681642, "stop": 1755207681736, "uuid": "f658e972-db4e-47b2-872a-a8466c2d5767", "historyId": "e3afb144354bc0591f14a81812069a10", "testCaseId": "e3afb144354bc0591f14a81812069a10", "fullName": "tests.test_get_order_by_track.TestGetOrderByTrack#test_get_order_invalid_track", "labels": [{"name": "feature", "value": "Get order by track"}, {"name": "epic", "value": "Orders API"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_get_order_by_track"}, {"name": "subSuite", "value": "TestGetOrderByTrack"}, {"name": "host", "value": "DESKTOP-SF5083A"}, {"name": "thread", "value": "23320-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_get_order_by_track"}], "titlePath": ["tests", "test_get_order_by_track.py", "TestGetOrderByTrack"]}